Fix:
error: implicit declaration of function 'dbf_load_file' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- src/main.c.orig	2012-01-05 14:28:21.000000000 -0600
+++ src/main.c	2022-05-19 03:48:31.000000000 -0500
@@ -33,6 +33,7 @@
 #endif
 
 #include "callbacks.h"
+#include "dbf.h"
 
 enum
 {
